File: core/src/main/java/com/rudderstack/android/sdk/core/RudderLibraryInfo.java
package com.rudderstack.android.sdk.core;

import com.google.gson.annotations.SerializedName;

class RudderLibraryInfo {
    @SerializedName("name")
    private final String name = BuildConfig.LIBRARY_PACKAGE_NAME;
    @SerializedName("version")
    private final String version = BuildConfig.VERSION_NAME;
}

File: android/src/main/java/com/rudderstack/android/models/RudderLibraryInfo.kt
package com.rudderstack.android.models

import com.google.gson.annotations.SerializedName

class RudderLibraryInfo(
    @SerializedName("name")
    private val name: String? = null,

    @SerializedName("version")
    private val version: String = "1.2.1",
)
